src/components/Breadcrumb/BreadcrumbSettings.js

```diff
@@ -0,0 +1,17 @@
+const breadcrumbSettings = [
```

Comment:
Ta opcja nie jest zła, jednak lepiej to zrobić tak jak w przypadku Boostrapa bo wtedy rzuca się to bardziej w oczy co zwiera dany element.